### PR TITLE
Task-5 AWS Simple Storage Service

### PR DESCRIPTION
**Pull request by agarmashevska**
Task 5 (AWS Simple Storage Service - S3)

##Evaluation criterias:

- [x] File serverless.yml contains configuration for importProductsFile function
- [x] The importProductsFile lambda function returns a correct response which can be used to upload a file into the S3 bucket
- [x] Frontend application is integrated with importProductsFile lambda (
FE pull request https://github.com/agarmashevska/nodejs-aws-fe/pull/3/files
)
Link to FE app - https://doi1o5vi7eolr.cloudfront.net
- [x] The importFileParser lambda function is implemented and serverless.yml contains configuration for the lambda

##Additional tasks:

- [x]  async/await is used in lambda functions
- [ ]  importProductsFile lambda is covered by unit tests (aws-sdk-mock can be used to mock S3 methods - https://www.npmjs.com/package/aws-sdk-mock)
- [x] At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into parsed folder, and then deleted from uploaded folder)

### CODE REVIEW

File: import-service/src/importFileParser.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            const Key = prop('s3.object.key', record)
+            const params = {
+                Bucket: BUCKET,
+                Key,
+            }
+            await importFile(s3, Key, params)
+        }
+
+        return {
+            statusCode: 200,
```
Comment (Shagon1k) [inline import-service/src/importFileParser.js]: Regarding specifications correct success status code for Event type lambdas is 202:
https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html#API_Invoke_ResponseSyntax

**Comment by OHalahan** [comment]
Please, add link to integrated FE, not just to PR

**Comment by agarmashevska** [comment]
> Please, add link to integrated FE, not just to PR

Sorry, forgot to add
https://doi1o5vi7eolr.cloudfront.net/admin/products

File: import-service/src/importFileParser.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                Key,
+            }
+            await importFile(s3, Key, params)
+        }
+
+        return {
+            statusCode: 202,
+            headers: corsHeaders,
+            body: JSON.stringify({
+               status: 202,
```
Comment (byslavik) [inline import-service/src/importFileParser.js:58]: I dont think that we really need to put status in body, as far as we can get it from response

**Review by byslavik — APPROVED** [review]

**Review by OHalahan — APPROVED** [review]

**Pull request merged**